zkevm-circuits/src/util/word.rs

```diff
@@ -408,12 +414,9 @@ impl<F: Field> Word<Expression<F>> {
         Word::new([self.lo() - rhs.lo(), self.hi() - rhs.hi()])
     }
 
-    /// Compress the lo and hi limbs into an expression without checking the overflow.
-    /// So far only use it for address.
-    /// TODO We should remove it before merging to the main branch.
-    #[deprecated(note = "no overflow check and unsafe. please consider keep word type")]
-    pub fn expr_unchecked(&self) -> Expression<F> {
-        self.lo() + self.hi() * (1 << (N_BYTES_HALF_WORD * 8)).expr()
+    /// No overflow check on lo/hi limbs
+    pub fn mul_unchecked(self, rhs: Self) -> Self {
```

Comment:
For a future PR I think we should try to remove `add_unchecked`, `sub_unchecked` and `mul_unchecked` and rewrite the gadgets that use these functionalities in a more clear way.